pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -61,13 +65,32 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 	timer := time.NewTimer(timeout)
 
 	e.reportRequiringApproval()
-	e.LogPersister.Info("Waiting for an approval...")
+	n, ok := e.MetadataStore.Stage(e.Stage.Id).Get(minApproverNumKey)
```

Comment:
Where do you put this `minApproveNum` at? And I'm guessing it's enough to fetch from `StageConfig`, isn't it?

Reply:
I mean I'm not sure why we need to save `minApproverNum` as metadata.

Reply:
You're right. Thanks.
Let me change it.